meta-data/Snakefile

```diff
@@ -69,6 +69,42 @@ def get_index_column(suffix):
     return "Gene ID"
 
 
+def filter_accessions_by_exp_type_and_species(wildcards):
+    from xml.dom import minidom
+    import csv
+    accessions = config["accessions"].split(",")
+    selected_accessions = []
+
+    for accession in accessions:
+        flag = False
+        xmldoc = minidom.parse(
+            config['cache_path']+"/data/"+accession+"/"+accession+"-configuration.xml"
+        )
+        exp_type = []
+        config_tag = xmldoc.getElementsByTagName("configuration")
+        for i in config_tag:
+            if "rnaseq" in i.getAttribute("experimentType"):
```

Comment:
experimentType can be
- 'rnaseq_mrna_baseline'
- 'rnaseq_mrna_differential'

Maybe you want to check here explicitly that the experiments belong to the first case?